charts/incubator/cryptpad/values.yaml

```diff
@@ -0,0 +1,78 @@
+#
+# IMPORTANT NOTE
+#
+# This chart inherits from our common library chart. You can check the default values/options here:
+# https://github.com/k8s-at-home/library-charts/tree/main/charts/stable/common/values.yaml
+#
+
+# config.js is derived from this map.
+# See: https://github.com/xwiki-labs/cryptpad/blob/main/config/config.example.js
+# httpUnsafeOrigin and httpSafeOrigin are automatically derived from the first and second host defined in the main
+# ingress, respectively.
+config:
+#  adminKeys:
+#    - "[cryptpad-user1@my.awesome.website/YZgXQxKR0Rcb6r6CmxHPdAGLVludrAF2lEnkbx1vVOo=]"
+  logLevel: info
+  verbose: false
+  logFeedback: false
+  logToStdout: true
+  installMethod: helm
+
+image:
+  # -- image repository
+  repository: promasu/cryptpad
+  # -- image tag (overrides appVersion)
+  tag: ""
+  # -- image pull policy
+  pullPolicy: IfNotPresent
+
+# -- environment variables. See more environment variables in the [cryptpad documentation](https://cryptpad.org/docs).
+# @default -- See below
+env:
+  # -- Set the container timezone
+  TZ: UTC
+  # Main and sandbox domains are automatically fetched from the first and second host defined in the ingress,
+  # respectively.
+  # These env vars are used in the internal nginx.
+  CPAD_MAIN_DOMAIN: "{{ (index .Values.ingress.main.hosts 0).host }}"
+  CPAD_SANDBOX_DOMAIN: "{{ (index .Values.ingress.main.hosts 1).host }}"
```

Comment:
This is again the same thing as the configmap: Attempt to be smart and not make the user input the same thing in two places.

Settings appear twice (as env vars and in the CM) because there are actually two different components reading these: The application itself, and an entrypoint script that generates an internal nginx config.